From: Mike Hommey <mh@glandium.org>
Date: Sat, 17 Jun 2017 07:17:24 +0900
Subject: Add ppc64el target for rust code

---
 build/moz.configure/rust.configure | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build/moz.configure/rust.configure b/build/moz.configure/rust.configure
index 156241f7f7db..e16d8bc9a4ed 100644
--- a/build/moz.configure/rust.configure
+++ b/build/moz.configure/rust.configure
@@ -174,6 +174,11 @@ def rust_triple_alias(host_or_target):
             ('sparc64', 'SunOS'): 'sparcv9-sun-solaris',
         }.get((host_or_target.cpu, os_or_kernel), None)
 
+        if rustc_target is None:
+            rustc_target = {
+		('ppc64', 'little', 'Linux'): 'powerpc64le-unknown-linux-gnu',
+            }.get((host_or_target.cpu, host_or_target.endianness, os_or_kernel))
+
         if rustc_target is None:
             die("Don't know how to translate {} for rustc".format(host_or_target.alias))
 
